### PR TITLE
Expose expiration in `stratis key list` output

### PR DESCRIPTION
**Pull request by waltdisgrace**
Addresses: https://github.com/stratis-storage/project/issues/200

Old output:
```
[gchin@gchin stratis-cli]$ ./bin/stratis key list
Key Description
test1          
test2          
test3
```
New output:
```
[gchin@gchin stratis-cli]$ ./bin/stratis key list
Key Description   Key Expiration
test1             No expiration 
test2             No expiration 
test3             No expiration
```

### CODE REVIEW

**Comment by waltdisgrace** [comment]
Rebased

**Comment by waltdisgrace** [comment]
I believe the Jenkins failure won't dissipate until https://github.com/stratis-storage/stratisd/pull/2154 gets merged:
```
KeyError: 'KeyringExpiration'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/fedora/workspace/stratis-cli/src/stratis_cli/_main.py", line 43, in the_func
    result.func(result)
  File "/home/fedora/workspace/stratis-cli/src/stratis_cli/_parser/_parser.py", line 87, in wrapped_func
    func(*args)
  File "/home/fedora/workspace/stratis-cli/src/stratis_cli/_actions/_top.py", line 818, in list_keys
    key_expiration = _fetch_expiration_property(proxy)
  File "/home/fedora/workspace/stratis-cli/src/stratis_cli/_actions/_top.py", line 181, in _fetch_expiration_property
    return _fetch_property(proxy, "KeyringExpiration")
  File "/home/fedora/workspace/stratis-cli/src/stratis_cli/_actions/_top.py", line 199, in _fetch_property
    return fetch_property(properties, property_name)
  File "/home/fedora/workspace/stratis-cli/src/stratis_cli/_actions/_utils.py", line 94, in fetch_property
    raise StratisCliPropertyNotFoundError(name)
stratis_cli._errors.StratisCliPropertyNotFoundError: The requested property 'KeyringExpiration' was not found in the FetchProperties result for a D-Bus object
```

File: src/stratis_cli/_actions/_top.py [hunk 1]
```diff
@@ -803,12 +815,19 @@ def list_keys(_):
...
+        ]
 
         print_table(
-            ["Key Description"], sorted(key_list, key=lambda entry: entry[0]), ["<"]
+            ["Key Description", "Key Expiration"],
+            sorted(rows, key=lambda entry: entry[0]),
+            ["<", "<"],
         )
 
+    #        print_table(["Key Expiration"], key_expiration, ["<"])
```
Comment (mulkieran) [inline src/stratis_cli/_actions/_top.py:829]: Please remove commented out code.

**Comment by waltdisgrace** [comment]
@mulkieran Would you like me to open a final PR after the Travis build succeeds?

**Comment by mulkieran** [comment]
> @mulkieran Would you like me to open a final PR after the Travis build succeeds?

@waltdisgrace Yes, please go ahead and do that. We'll have to hold the PR in pending for a while, but I think it is ready for final review.

**Pull request closed without merging**